checkov/common/bridgecrew/platform_integration.py

```diff
@@ -361,6 +364,12 @@ def setup_http_manager(self, ca_certificate: str | None = None, no_cert_verify:
                 )
         logging.debug('Successfully set up HTTP manager')
 
+    @staticmethod
+    def set_config():
+        # asyncio - on windows aiodns needs SelectorEventLoop
+        if sys.platform == 'win32':
+            asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
```

Comment:
@AdamDev How does it solve the issue?

Reply:
This issue exists in many places in the network, this is the suggested solution:
https://github.com/saghul/aiodns/issues/86#issuecomment-1674906449

Root cause:
https://github.com/saghul/aiodns/issues/86#issuecomment-1437115620